lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-redhat-infra_manager.rb

```diff
@@ -1,4 +1,6 @@
 module MiqAeMethodService
   class MiqAeServiceManageIQ_Providers_Redhat_InfraManager < MiqAeServiceEmsInfra
+    expose :validate_import_vm
+    expose :import_vm
```

Comment:
@gmcculloug I don't believe so, If the import operation is started successfully there will be no useful return value and in case of an error an exception is thrown. So I think using the `:override_return` facility is the way to go here.